plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -138,7 +136,7 @@ export class PgInstrumentation extends InstrumentationBase {
         }
 
         const span = plugin.tracer.startSpan(
-          `${PgInstrumentation.COMPONENT}.connect`,
+          `connect${this.database ? ` ${this.database}` : ''}`,
```

Comment:
Here we are doing 
```js
`connect${this.database ? ` ${this.database}` : ''}`
```

and below we have 
```js
`connect ${this.options.database}`,
```

and in utils.ts we have:
```js
`${command}${dbName ? ` ${dbName}` : ''}`
```

Perhaps we can extract this logic into a function in utils that receives command and dbName and return a db span name from these values? and then use it in all 3 places?